packages/react-scripts/fixtures/kitchensink/src/vendors.js

```diff
@@ -0,0 +1 @@
+module.exports = ['react', 'react-dom'];
```

Comment:
Why isn't this using ES6 exports?

Reply:
If this is supposed to be static, can it be a JSON file instead?

Reply:
It could be a JSON file, but you cannot add comments in a JSON file
and it's with regular exports because it's read by node directly and not by webpack